hudi-flink/src/main/java/org/apache/hudi/sink/StreamWriteOperatorCoordinator.java

```diff
@@ -137,6 +149,8 @@ public void start() throws Exception {
     reset();
     // writeClient
     this.writeClient = StreamerUtil.createWriteClient(conf, null);
+    // table state
```

Comment:
Useless comment?

Reply:
Would like to keep it.

Reply:
The meaning of the comment is to facilitate the explanation of the code logic. I think there is no need to add these comments here, because it will not help others understand this logic.